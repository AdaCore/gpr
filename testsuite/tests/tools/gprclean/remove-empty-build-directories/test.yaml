description: S513-047 imported test (no warning on gprclean -p)
driver: python_script
