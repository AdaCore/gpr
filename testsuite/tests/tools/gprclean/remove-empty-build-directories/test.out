using project file /s.gpr
s.gpr: warning: there are no sources of language "Ada" in this project
